src/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "allowJs": true,
     "checkJs": true,
-    "lib": ["es2018", "dom"],
+    "lib": ["es2018", "dom", "dom.iterable"],
```

Comment:
Curious, is https://github.com/microsoft/TypeScript/tree/main/lib a reasonable source of available `lib`s, or is there a more useful source? What was your process of working out that this was what was needed to support `URLQueryParams` (which is iterable)?

Reply:
I got confused at first and then googled for the solution, which pointed me to an issue in the TypeScript repo on GitHub.